### PR TITLE
Pre-sprint evaluation cleanup

### PR DESCRIPTION
**Pull request by alexsenden**
### Changes:
- #65 
- #66 
- #67 
- #69 
- #21
- #15 
- Added seed data
- Replaced css with inline styles and styled components
- Fixed a lot of bugs
- Made a lot of things dynamic
- Added meeting minutes

We should start off next sprint by doing some refactoring, and focusing hard on testing.

### CODE REVIEW

File: harmony-api/src/services/postService.ts [hunk 1]
```diff
@@ -88,6 +111,9 @@ export const validateBody = (postData: Post): Array<string> => {
   if (!postData.body) {
     return ['body field is required to create a new discussion or review post']
   }
+  if (postData.title.length > 2000) {
+    return ['title field must be less than 1000 characters']
```
Comment (EddenWong) [inline harmony-api/src/services/postService.ts:115]: This should be for the body right?

```suggestion
  if (postData.body.length > 2000) {
    return ['body field must be less than 1000 characters']
```

**Review by EddenWong — CHANGES_REQUESTED** [review]
Only found the one thing. Other than that, looks good to me 👍

**Comment by Developik** [comment]
Looks good to me as well

**Review by EddenWong — APPROVED** [review]
Looks all good to me 🥇

**Pull request merged**